README.markdown

```diff
@@ -50,7 +50,6 @@ This explanation assumes you're familiar with developing Ruby on Rails applicati
 
 | Toggle in development | command                      | default  |
 | ---------             | -------                      | -------  |
-| Spring                | `bin/rake dev:spring`        | disabled |
```

Comment:
This PR drops Spring since [bootsnap](https://blog.bigbinary.com/2018/01/01/rails-5-2-adds-bootsnap-to-the-app-to-speed-up-boot-time.html) replaces it.